core_impl/src/test/java/io/opencensus/impl/tags/TagContextDeserializationTest.java

```diff
@@ -15,60 +15,54 @@
 
 import static com.google.common.truth.Truth.assertThat;
 
-import io.grpc.Context;
-import io.opencensus.common.Scope;
-import io.opencensus.internal.SimpleEventQueue;
+import com.google.common.collect.Lists;
 import io.opencensus.internal.VarInt;
-import io.opencensus.testing.common.TestClock;
+import io.opencensus.tags.TagContext;
+import io.opencensus.tags.TagContexts;
+import io.opencensus.tags.TagKey.TagKeyString;
+import io.opencensus.tags.TagValueString;
 import java.io.ByteArrayInputStream;
 import java.io.ByteArrayOutputStream;
 import java.io.IOException;
 import java.io.InputStream;
-import java.util.HashMap;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 
-/** Tests for {@link StatsContextFactory}. */
+/**
+ * Tests for deserializing tags with {@link SerializationUtils} and {@link
+ * TagContextSerializerImpl}.
+ */
 @RunWith(JUnit4.class)
 public class TagContextDeserializationTest {
```

Comment:
I think the class would be too large if we combined all of the tests, and serialization vs deserialization seemed like a good way to split it.  The test suites will also grow when we add support for the other tag types.